src/components/reviews/reviewsList.jsx

```diff
@@ -7,30 +7,24 @@ class ReviewsList extends React.Component {
     super(props);
 
     this.state = {
-      reviewsShown: 0,
-      showMoreButton: true,
+      reviewsShown: 2,
       filters: new Set()
     };
-    this.loadMoreBound = this.loadMore.bind(this);
+    this.changeLoadBound = this.changeLoad.bind(this);
     this.toggleFilterBound = this.toggleFilter.bind(this);
     this.clearFiltersBound = this.clearFilters.bind(this);
   }
 
   componentDidMount() {
-    // get metadata dispatch
-    // get reviews dispatch
-    // set state reviewsShown to min of 2 or length of reviews
-    // if length of reviews <= 2,
-      // set state showMoreButton to false
+    // fetch reviews and metadata from API
+    this.props.getData(3); // HARD-CODED FOR NOW
```

Comment:
I have the same thing, let me know if you find an easily implemented way to pull the ID from the store once Overview has fetched it